### PR TITLE
Updated progressbar gif and README

### PR DESCRIPTION
**Pull request by iRhonin**

### CODE REVIEW

File: examples/progressbar.py [hunk 1]
```diff
@@ -1,30 +1,21 @@
-import time
+from time import sleep
```
Comment (pylover) [inline examples/progressbar.py]: Is this file a gif? 

Or you must send two pull requests with separate title.

**Review by pylover — CHANGES_REQUESTED** [review]
The title does not completely describe the changeset.

Please be more cut and clean.

File: examples/progressbar.py (hunk 1 above)
Comment (iRhonin) [reply examples/progressbar.py]: No, you are right, I will be sending another pull request for it.

**Comment by iRhonin** [comment]
@pylover Done.

**Review by pylover — CHANGES_REQUESTED** [review]
The gif image has a very low quality.

**Pull request merged**